src/main.c

```diff
@@ -132,6 +131,8 @@ int main(int argc, char *argv[]) {
     } else if(rec->count > 0) {
         init_flags.record = 1;
         strncpy(init_flags.rec_file, rec->filename[0], 254);
+    } else if(renderer->count > 0) {
```

Comment:
Might be better to parse the input here, and throw an error if the renderer is unknown/unavailable?

Reply:
Well, that would duplicate things. Currently, if I run `/openomf --renderer foobar`, I get
```
[      0][D] video_scan_renderers(): Renderer 'OpenGL3' is available
[      0][D] video_scan_renderers(): Renderer 'NULL' is available
[      0][E] video_find_renderer(): Unable to find specified renderer 'foobar', trying other alternatives ...
[      0][I] video_find_renderer(): Found available renderer 'OpenGL3'!
```
So in this sense, all is fine already.